arelle/plugin/validate/EDINET/rules/upload.py

```diff
@@ -656,6 +658,31 @@ def rule_EC1007E(
                     line=elt.sourceline,
                     modelObject=elt,
                 )
+                continue
+            path = Path(value)
+            if len(path.parts) < 2:
```

Comment:
Is this the actual intent?  I would have guessed it  "don't link to thinks outside the filing".  For example, given `filingDir/subdir/file.htm`, I would have thought `other.htm` is ok (reference file in same directory), `../top.htm` is ok (referencing a file in a parent dir but still within the file), `../subdir2/other2.htm` is ok, but `../../outside.htm` and `/outside.htm` are not ok.

(If it is actually the intent, I guess `./other.htm` is accepted but shouldn't be?)